cypress/e2e/views/common.view.ts

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 export const itemsPerPageToggleButton = "button[aria-label='Items per page']";
-export const submitButton = "#identity-form-submit";
+export const submitButton = "button[aria-label=submit]";
```

Comment:
**chore:** `submitButton` is a widely used constant, if you saw the id changed in only some places (like businessService creation) update the value only in those places.

And try to use IDs wherever you can, they're more reliables than aria-labels

Reply:
This button is not used anywhere ? 
If you see any submit button with different ID or missing ID , you can raise bug and ask them to make it standard as other submit buttons .

Reply:
i see that the submit button id has changed at each form, but the id always contains "form-submit"
 so i initialize the element with that sub string.

Reply:
form-submit id substring is unique only for submit button